update: update the Compose Multiplatform and Jetpack Compose article (#343)
JetBrains/kotlin-multiplatform-dev-docs: topics/compose/compose-multiplatform-and-jetpack-compose.md

[//]: # (title: Compose Multiplatform and Jetpack Compose)

> ![Compose Multiplatform created by JetBrains, Jetpack Compose created by Google](og-image-compose-multiplatform-and-jetpack-compose.png)

Reviewer: ```suggestion
![Compose Multiplatform created by JetBrains, Jetpack Compose created by Google](compose-multiplatform-and-jetpack-compose.png){width=730}
```
Fixes the file location and scales the image a bit.

<tldr>
This article explains the relationship between Compose Multiplatform and Jetpack Compose.

> It covers shared concepts like composables, state management, and UI components, along with platform-specific features and limitations.

Reviewer: ```suggestion
```
I think this sentence can be removed to make the tldr shorter without losing any essential information from the article summary. WDYT?

allowing you to write shared compose UI code that can run on Android, iOS, desktop, and web:

|                  | **Compose Multiplatform**  | **Jetpack Compose** |
|------------------|----------------------------|---------------------|
| **Platforms**    | Android, iOS, desktop, web | Android             |
| **Supported by** | JetBrains                  | Google              |

## Jetpack Compose and composables

Jetpack Compose is a declarative UI toolkit for building native Android interfaces.
Its foundation is _composable_ functions, marked with the `@Composable` annotation.
These functions define parts of the UI and automatically update when the underlying data changes.
You can combine composables to build layouts, handle user input, manage state, and apply animations.
Jetpack Compose includes common UI components like `Text`, `Button`, `Row`, and `Column`, which you can customize with modifiers.

Compose Multiplatform builds on the same principles. 
It shares the Compose compiler and runtime with Jetpack Compose and uses the same APIs — `@Composable` functions,
state management tools like `remember`, layout components, modifiers, and animation support.

> This means you can reuse your Jetpack Compose knowledge to build cross-platform UIs for Android, iOS, desktop, and web.

Reviewer: ```suggestion
This means you can reuse your Jetpack Compose knowledge with Compose Multiplatform to build cross-platform UIs for Android, iOS, desktop, and web.
```

## Compose Multiplatform and Jetpack Compose features

> > If you're experienced with Android development, you can apply your knowledge of Jetpack Compose when working with Compose Multiplatform.

Reviewer: ```suggestion
```
Maybe remove this sentence as it's very similar to the paragraph above?

> > You can also learn about fundamentals of the both UI frameworks from almost any Jetpack Compose material,

Reviewer: ```suggestion
> You can learn about the fundamentals of both UI frameworks from almost any Jetpack Compose material,
```
